W/WrapIt/build_tarballs.jl

```diff
@@ -110,15 +140,20 @@ cp -p "../artifacts/$llvm_uuid/lib/$llvmlib" lib/
 install_license "${WORKSPACE}/srcdir/wrapit/LICENSE"
 """
 
-# LLVM_jll not available for i686+musl (see https://github.com/JuliaPackaging/Yggdrasil/blob/f756bd9eb500ad68a8b8bb4413d6692c8e766a47/L/LLVM/common.jl)
-# Windows not supported:
-platform_veto(p) = Sys.iswindows(p) || (arch(p) == "i686" && libc(p) == "musl")
+# Windows is not supported.
+# lidation done on 2025-02-20 leads to failure for x86_64-apple-darwin (issue with 'rm -r' 
+# from the recipe used to add std::filesystem support) => vetor this arch.
+# 2025-03-03: following arch vetoed because of validation failure:
+#  - x86_64-apple-darwin: I/O error when running the recipe to add std::filesystem support (was working before)
```

Comment:
You need a newer SDK

Reply:
The two sentences you wrote are conflicting with each other.

Reply:
Also, you don't need 11.3 (otherwise aarch64-darwin wouldn't work either), 10.15 is sufficient:
https://github.com/JuliaPackaging/Yggdrasil/blob/6e301924da9dbb4b19ce9a424842315b3f7fdf87/C/Ccache/build_tarballs.jl#L21-L29

Reply:
And if you can't delete files from the sysroot it's a problem with your setup (it depends on how the filesystem is mounted, apparently it's read-only), it'd work fine here.